app/assets/js/mautic-form-src.js

```diff
@@ -220,6 +220,22 @@ var t,e;t=this,e=function(){"use strict";function t(t,e){var n=Object.keys(t);if
                     Form.prepareShowOn(formId);
                     Form.preparePagination(formId);
 
+                    const theForm = forms[i];
+                    const hasPageBreak = theForm.querySelectorAll('[data-mautic-form-page]').length > 1;
+                    if (hasPageBreak) {
+                        const inputs = theForm.querySelectorAll('input, select, button');
+                        Array.prototype.forEach.call(inputs, function(input) {
+                            if (input.tagName.toLowerCase() !== 'textarea') {
+                                input.addEventListener('keydown', function(e) {
+                                    if (e.key === 'Enter' || e.keyCode === 13) {
```

Comment:
Consider using only `e.key === 'Enter'` here, as `keyCode` is deprecated and its support depends on the browser engine.
```suggestion
                                    if (e.key === 'Enter') {
```